config/application.rb

```diff
@@ -19,7 +19,8 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
+    #config.time_zone = 'Central Time (US & Canada)'
+    ENV['TZ']='UTC'
```

Comment:
could this be in `config/environments/test.rb` ?  I moved if there and your test (and all the others) still passed.  We only need this for the test environment no?